### ISSUE TITLE
Speed increases when size is changed

### ISSUE THREAD

**Issue by ghost** [huskeee/tkvideo#1]
The speed of the video increases when you reduce the size of the video. How to fix this? Referring to tkVideo module. Really useful.

**Comment by huskeee** [comment]
What are the steps to reproduce the problem? Thanks for your kind comments!

p.s: My answers are late due to a hectic university schedule :(

**Comment by coenfuse** [comment]
Whenever the video's size is reduced, the video speeds up proportionally. And whenever the size is increased, the video slows down accordingly. I couldn't find a the resolution where the video plays at 1x.

This is the following syntax I'm using in my project
`player = tkvideo.tkvideo("clip.mp4", video_label, loop = 1, size = (1280,720))`
`player.play()`

Yesterday this issue was random, but today. It is just persistent. I couldn't make it work.

Awaiting you response.

**Comment by huskeee** [comment]
It seems like the speed is connected proportionally to the resolution. It's probably caused by this for loop:
```py
 for image in frame_data.iter_data():
                    frame_image = ImageTk.PhotoImage(Image.fromarray(image).resize(self.size))
                    label.config(image=frame_image)
                    label.image = frame_image
```
I really don't know how to resolve this issue, as this was a sideproject for a former uni assignment, and the university I'm in now teaches us C++.

**Comment by TheLizzard** [comment]
This library doesn't keep track of the time taken for each frame to be shown. Therefore, the speed of the video depends on the speed of the computer and the number of other tasks that are running on that computer.

Also as a side note: this library uses threading and tkinter at the same time even though tkinter isn't thread safe. This can cause errors like [this](https://stackoverflow.com/q/14694408/11106801).

**Comment by huskeee** [comment]
Closed with PR #8 .
